### ISSUE TITLE
27 Github action tests fails out of 53 test for the sprint 33 snapshots 

### ISSUE THREAD

**Issue by xdxxxdx** [Informatievlaanderen/VSDS-LDES-E2E-testing#364]
**Validation tags:**
 - [ldes/ldes-server:2.0.0-SNAPSHOT](https://hub.docker.com/r/ldes/ldes-server/tags)
 - [ldes/ldi-workbench-nifi:1.4.0-SNAPSHOT](https://hub.docker.com/r/ldes/ldi-workbench-nifi/tags) 
 - [ldes/ldi-orchestrator:1.4.0-SNAPSHOT](https://hub.docker.com/r/ldes/ldi-orchestrator/tags)

**Scenario:**
Run the Github action against the provided tag: https://github.com/xdxxxdx/VSDS-LDES-E2E-testing/actions/runs/5900026842

**Current result:**
27/53 failed.
![image](https://github.com/Informatievlaanderen/VSDS-LDES-E2E-testing/assets/15314807/85fbe7f6-c79e-4982-a957-d41c3e1e6e70)

**Expected result:**
All tests should pass.
Or there is an issue in the test setup.
Or there is an issue in the VSDS solution.

**Comment by rorlic** [comment]
The tests have been written with some knowledge on how things work internally (e.g. fragmentation happens during ingest). The LDES server and other LDES components sometimes change their inner workings (e.g. fragmentation now happens after ingest) which breaks the tests sometimes. In addition, some fragmentations have been changed (e.g. pagination is now done by default).

As we are also working on onboardings so the E2E tests are not always updated immediately.

I have disabled 4 tests and fixed the server basic and fragmentation tests.
There are still 12 out of the 49 tests that fail (mostly due to some config issues). I will try to fix these soon.
